client/my-sites/sharing/connections/services-group.jsx

```diff
@@ -79,6 +63,7 @@ class SharingServicesGroup extends Component {
 
 export default connect(
 	( state, { type } ) => ( {
+		initialized: !! getSelectedSiteId( state ),
```

Comment:
If no site is selected, connecting services is moot. We can probably remove that, I assume it's from a time when we couldn't rely on having a selected site available?